zui/src/ui/Form.tsx

```diff
@@ -11,6 +11,7 @@ export type ZuiFormProps<UI extends UIComponentDefinitions = DefaultComponentDef
   onChange: (value: any) => void
   disableValidation?: boolean
   fallback?: BoundaryFallbackComponent
+  dataTransform?: (data: any) => any
```

Comment:
:thought_balloon: **thought:** it's not really clear what transformation happens here. Are we transforming the input value or the output value? and how is it used? Might be useful to add descriptions to properties just to be clearer

any is being used in a lot of places, what do you think about adding "fake" custom types to make usage a bit clearer across the component? 
ex: 
```
type InputValue = any
type OutputValue = any
type FormData = any
```